Dotnet.Homeworks.Features/Cqrs/UserManagement/DirectoryReference.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Dotnet.Homeworks.Features.Cqrs.UserManagement;
+
+/// <summary>
+/// Static class that returns namespace of the feature
+/// </summary>
+public static class DirectoryReference 
+{
+    public static readonly string? Namespace = typeof(DirectoryReference).Namespace;
```

Comment:
```suggestion
    public static readonly string Namespace = typeof(DirectoryReference).Namespace!;
```
согласно summary этого свойства в нашем случае не может быть null: _null if the current instance has no namespace or represents a generic parameter._